core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -290,11 +290,11 @@ object GameStarter {
 
             
             if (civ.isCityState()) {
+                // City states should spawn with one settler only irregardless of era and difficulty
```

Comment:
Please don't use the word irregardless

Reply:
:rofl: I just had to check this discussion because I saw that creative word in Studio without the spellchecker whining - and - eagle eyes. I actually quite like it.